config/channels/in-memory-channel-ns/README.md

```diff
@@ -0,0 +1,39 @@
+# Namespace-Scoped In-Memory Channels
+
+Namespace-scoped in-memory channels share the same characteristics as the
+[in-memory channels](../in-memory-channel/README.md). The only difference is the in-memory
+dispatcher is installed in the same namespace as the channel.
+
```

Comment:
Can cluster-scoped ../in-memory-channel and in-memory-channel-ns be installed both in the same cluster?

Reply:
After `ko apply -f config/channels/in-memory-channel` it seems I get both dispatchers?

```
(base) aslom@m eventing (pr/lionelvillard/2555) $ kubectl get deployment -n knative-eventing
NAME                  READY   UP-TO-DATE   AVAILABLE   AGE
eventing-controller   1/1     1            1           10d
eventing-webhook      1/1     1            1           10d
imc-controller        1/1     1            1           10d
imc-dispatcher        1/1     1            1           10d
(base) aslom@m eventing (pr/lionelvillard/2555) $ kubectl get deployment -n default
NAME                                                           READY   UP-TO-DATE   AVAILABLE   AGE
hello-display                                                  1/1     1            1           9d
imc-dispatcher                                                 1/1     1            1           114s
```

Reply:
this is not allowed. Adding it the doc. Thanks :-)